ansible/kubernetes.yml

```diff
@@ -1,6 +1,31 @@
----
+- hosts: k8s-cluster
+  tasks:
+    - set_fact:
+        kubeconfig_localhost: true
+        artifacts_dir: "{{ inventory_dir }}"
+
 - import_playbook: roles-external/kubespray/cluster.yml
 
+- name: 'Bringing kubeconfig.dec in place'
+  hosts: k8s-cluster
+  become: no
+  tasks:
+    - delegate_to: localhost
+      block:
+      - name: "Checking if 'admin.conf' file still exists"
+        stat:
+          path: "{{ artifacts_dir }}/admin.conf"
+        register: file_adminconf
+      - when: file_adminconf.stat.exists
+        block:
+        - name: 'Renaming kubeconfig file provided by Kubespray'
+          copy:
+            src: "{{ artifacts_dir }}/admin.conf"
+            dest: "{{ artifacts_dir }}/../kubeconfig.dec"
+        - file:
+            path: "{{ artifacts_dir }}/admin.conf"
+            state: absent
+
```

Comment:
@arianvp this probably breaks an assumption for in the offline package, doesnt it? 

Note, this is the first step (out of two) to automate sopsing kubeconfig in an $ENV_DIR when bootstrapping a K8s cluster.